### ISSUE TITLE
Streamline steps in the Filebeat Getting started guide

### ISSUE THREAD

**Issue by dedemorton** [elastic/beats#10640]
Input from Ron Cohen:

We recently discussed ideas to make the getting started steps even simpler/shorter.

For example, as a new-comer, it could look like setting up Logstash is part of the necessary or recommended workflow. We could perhaps move the Logstash part out so it's not a step?

More details about required changes to be added later...

**Comment by dedemorton** [comment]
Capturing some other things I want to fix:

- [ ] Make the logstash config optional.
- [ ] Users should load the Beats assets (dashboards, pipelines, etc) before changing the output configuration to LS.

**Comment by dedemorton** [comment]
More thoughts on the notion of having a Quick Start Guide. What if we were to have a one-page quick start guide that distills the content down to just what you need to know to get started evaluating Beats. Then we could move the other content to "Setting up and running" and add more detail, where required, without having the getting started cover all the things.

@bmorelli25 Curious to hear your thoughts on this ^^.

**Comment by bmorelli25** [comment]
> We've used the getting started guide to teach some basic concepts (what is an index, what are dashboards, etc), but that means users need to read a lot of detail that is not really required just to get started.

I see why that information is included in the Getting Started section, but I don't think it makes the most sense. I think your _Quick Start Guide_ solution is spot on. Simplify and get the consumer up and running as easily as possible.

**Comment by roncohen** [comment]
absolutely agree regarding "Quick Start Guide" versus "Getting started Guide" 👍 
I would make the quick start guide as short as possible and include links from there to more information for users who want to do something out of the ordinary.

**Comment by roncohen** [comment]
Any progress on this? Let me know if i can help

**Comment by dedemorton** [comment]
@roncohen Not yet...really sorry. Unfortunately, I keep needing to bump the priority of this work in favor of other things on the doc roadmap. It would be *very* easy for me to remove Logstash as a main step in the getting started  (would take a few minutes). If it helps, I can move that forward now.

**Comment by dedemorton** [comment]
Part of this effort involves restructuring the Beats docs so we have someplace where the extra content (stuff that doesn't belong in the GS) lives. To kick off this effort, I've started restructuring the Beats docs by dividing the config content into two sections: reference and how-to. After I've finalized the PR, I'll work on ripping out the excess content and moving it to the how-to section, as appropriate.  See in-progress PR for the restructuring effort: https://github.com/elastic/beats/pull/16825
